app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt

```diff
@@ -634,7 +637,7 @@ class MainActivity : SimpleActivity() {
 
         val faqItems = arrayListOf(
             FAQItem(R.string.faq_1_title_commons, R.string.faq_1_text_commons),
-            FAQItem(R.string.faq_1_title, R.string.faq_1_text)
+            FAQItem(R.string.faq_1_title, R.string.faq_1_text),
```

Comment:
these commas arent needed

Reply:
ktlint responsible for the formatting with trailing commas during commit, I will disable it.